### PR TITLE
[Feature] Adding PWC metafile

### PR DESCRIPTION
**Pull request by LXXXXR**
Hi, 

In this pull request, metafiles are added for PWC support.
Pls kindly take a look. Thank you.

### CODE REVIEW

File: configs/fp16/metafile.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Collections:
+  - Name: ResNet
```
Comment (ZwwWayne) [inline configs/fp16/metafile.yml:2]: This should already be defined in resnet?

File: configs/resnet/metafile.yml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    Paper: https://openaccess.thecvf.com/content_cvpr_2016/html/He_Deep_Residual_Learning_CVPR_2016_paper.html
+    README: configs/resnet/README.md
+
+Models:
+- Config: configs/resnet/resnet18_b16x8_cifar10.py
+  In Collection: ResNet
+  Metadata:
+    FLOPs: 0.56 G
+    Parameters: 11.17 M
+    Training Data: Cifar-10
```
Comment (ZwwWayne) [inline configs/resnet/metafile.yml:22]: Cifar or CIFAR? May be consistent with PWC https://paperswithcode.com/datasets

File: configs/resnet/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    Training Resources: 8x 1080 GPUs
+    Epochs: 200
+    Batch Size: 128
+  Name: ResNet-18-b16x8
```
Comment (ZwwWayne) [inline configs/resnet/metafile.yml:26]: name could use similar name as config, thus it can be distinguished easier and will not be duplicated.

**Comment by ZwwWayne** [comment]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Review by ycxioooong — APPROVED** [review]
LGTM

**Pull request merged**